bin.js

```diff
@@ -44,23 +44,31 @@ if (cmd === 'listen') {
   server.listen(argv.port, argv.host, function () {
     console.log('signalhub listening on port %d', server.address().port)
   })
-  return
+  process.exit(0)
```

Comment:
this unfortunately won't work since that kills the process and therefor the server :(